src/ssh/azext_ssh/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+    "azext.isPreview": true
```

Comment:
Better do define a `minCLICoreVersion` here. You can reference [here](https://github.com/Azure/azure-cli-extensions/blob/e99c1374cacb559bcf8883147eaae5a6df0118c6/src/stream-analytics/azext_stream_analytics/azext_metadata.json#L3).